x-pack/tsconfig.json

```diff
@@ -13,10 +13,7 @@
     "plugins/apm/e2e/cypress/**/*",
     "plugins/apm/scripts/**/*",
     "plugins/licensing/**/*",
-    "plugins/global_search/**/*",
-    "../src/plugins/usage_collection/**/*",
-    "../src/plugins/telemetry_collection_manager/**/*",
-    "../src/plugins/telemetry/**/*"
+    "plugins/global_search/**/*"
```

Comment:
self-review: I deleted the `usage_collection`, `telemetry_collection_manager` and `telemetry` plugin paths from "exclude" because they're not referenced in the "include" array. "exclude" only applies to directories listed in the "include" array and there isn't a path to "../src/plugins".